src/custom-element.js

```diff
@@ -559,16 +559,20 @@ function createBaseCustomElementClass(win) {
      * Updates the layout box of the element.
      * See {@link BaseElement.getLayoutWidth} for details.
      * @param {!./layout-rect.LayoutRectDef} layoutBox
+     * @param {boolean=} opt_hasMeasurementsChanged
```

Comment:
Nit: Should be "have". 😛  How about `opt_measurementsChanged`?

Reply:
opt_measurementsChanged sounds good. Thanks